tutorial/debezium-cassandra-init/log4j.properties

```diff
@@ -0,0 +1,6 @@
+log4j.rootLogger=INFO, A1
+
+log4j.appender.A1=org.apache.log4j.ConsoleAppender
+
+log4j.appender.A1.layout=org.apache.log4j.PatternLayout
+log4j.appender.A1.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
```

Comment:
There are some new-lines at EOF missing in this PR, here and elsewhere.

Reply:
No, I don't think any lines are missing. This file contains this much only.

Reply:
@ani-sha Gunnar means the line end (LF char) at the last line of the file. Not that any text is missing